properties.py

```diff
@@ -469,11 +473,11 @@ def reset_ll_object_index(self, context):
         min=-32, max=32, default=-1)
     max_trace_depth = IntProperty(
         name="Max Trace Depth",
-        description="Maximum number of ray bounces (0 disables ray tracing)",
+        description="Maximum number of times a ray can bounce before termination.\nLower settings will render faster, but may also dull bounce lighting.",
```

Comment:
Lightyear and I were having a discussion on Slack about this.  It actually might be advantageous for you to not merge the tooltips.  I think I jumped the gun.